utils/sf-config.js

```diff
@@ -7,57 +7,101 @@
 
 const { dirname } = require('path');
 const { cosmiconfigSync } = require('cosmiconfig');
-const { isMultiPackageProject } = require('./project-type');
+const { isPlugin } = require('./project-type');
 
 const PACKAGE_DEFAULTS = {
   scripts: {
-    build: 'sf-build',
+    build: 'wireit',
     clean: 'sf-clean',
     'clean-all': 'sf-clean all',
-    compile: 'sf-compile',
+    compile: 'wireit',
     docs: 'sf-docs',
-    format: 'sf-format',
-    // Cleaner errors than ts-node through tests
-    pretest: 'sf-compile-test',
-    test: 'sf-test',
-    lint: 'sf-lint',
+    format: 'wireit',
+    // this will be removed
+    pretest: undefined,
+    posttest: undefined,
+    test: 'wireit',
+    'test:compile': 'wireit',
+    'test:only': 'wireit',
+    lint: 'wireit',
     prepack: 'sf-prepack',
   },
-};
-
-const LERNA_DEFAULTS = {
-  scripts: {
-    build: 'lerna build',
-    clean: 'lerna clean',
-    'clean-all': 'lerna sf-clean all',
-    compile: 'lerna compile',
-    docs: 'lerna docs',
-    format: 'lerna format',
-    test: 'lerna test',
-    lint: 'lerna lint',
+  wireit: {
+    build: {
+      dependencies: ['compile', 'lint'],
+    },
+    compile: {
+      command: 'tsc -p . --pretty --incremental',
+      files: ['src/**/*.ts', 'tsconfig.json', 'messages/**'],
+      output: ['lib/**', '*.tsbuildinfo'],
+      clean: 'if-file-deleted',
+    },
+    format: {
+      command: 'prettier --write "+(src|test|schemas)/**/*.+(ts|js|json)|command-snapshot.json"',
+    },
+    lint: {
+      command: 'eslint src test --color --cache --cache-location .eslintcache',
+      files: ['src/**/*.ts', 'test/**/*.ts', 'messages/**', '.eslintignore', '.eslintrc.js'],
+      output: [],
+    },
+    'test:compile': {
+      command: 'tsc -p "./test" --pretty',
+      files: ['test/**/*.ts', 'tsconfig.json', 'test/tsconfig.json'],
+      output: [],
+    },
+    test: {
+      dependencies: ['test:only', 'test:compile'],
+    },
+    'test:only': {
```

Comment:
nit: I suppose `only` was added because `test` is already used to group deps, could this be renamed to `test:run`?

Reply:
yeah.  I think there were some places we had test:only currently (one was toolbelt) 
https://github.com/salesforcecli/toolbelt/blob/34e3d0ede4cdd22bbd37102edbda14d89e233417/package.json#L167

But I'm good either way.